statuslight_control/.tool-versions

```diff
@@ -0,0 +1 @@
+elixir 1.10.3
```

Comment:
Nonblocking - I think I meant to have .tool-versions in my .gitignore.  I am not sure what community preference on this is generally but I typically would rather not check this in.

Reply:
Certainly open to counter-opinion on this, too, or just ignore and leave it since my comment is fully nonblocking :)

Reply:
We always checked them in on the pro-flow team, it just helps since my default elixir is set to 1.9.x and I was getting errors mix compiling until I manually swapped elixir versions since in the mix.exs it was set to elixir >= 1.10.x, I don't have a strong preference whether or not they're checked in, they can be .gitignored